docs/tests/workshop.md

```diff
@@ -0,0 +1,38 @@
+# workshop
+The workshop type is being used for creating on cluster, throwaway postgres/redis instances for troubleshooting. 
+While this isn't functionality used in production, there are some manual test cases and sops that rely on it.
+
+## Usage
+To create a workshop instance, log in to a cluster and issue one of the following commands:
+- Postgres
```

Comment:
The lack of extra line above is causing this bullet point not to be rendered by mkdocs. I also think this reads better if it a heading and not a bullet point.
```markdown
### Postgres
```